src/main/java/com/example/security/JWTAuthenticationFilter.java

```diff
@@ -12,6 +12,13 @@
 
 public class JWTAuthenticationFilter extends GenericFilterBean {
 
+    private TokenAuthenticationService tokenAuthenticationService;
+
+    @Autowired
```

Comment:
I had to change it like that and generally make it working differently because in TAS I am returning Authenticated User much more filled than your simplified, that's why.